Gemfile

```diff
@@ -9,8 +9,9 @@ gem 'sass-rails', '~> 5.0'
 # Use Uglifier as compressor for JavaScript assets
 gem 'uglifier', '~> 4.1'
 
-# pinning myql2 since 0.5.3 is currently not compatible with our version of CentOS and Ruby (as of Apr 2020)
-gem 'mysql2', '~> 0.5', '< 0.5.3'
+# mysql2 0.5.3 is not compatible with older versions of CentOS and Ruby (as of Apr 2020)
```

Comment:
We're not on centos anymore are we? Can we drop this comment?

Reply:
dropped comment